lib/outputs/models.py

```diff
@@ -0,0 +1,170 @@
+from dataclasses import dataclass
+
+from config.config import timestamp
+from lib.check.models import Check_Report, Organizations_Info
+
+
+@dataclass
+class Compliance_Framework:
+    Framework: str
+    Version: str
+    Group: list
+    Control: list
+
+
+@dataclass
+class Check_Output:
+    STATUS: str
+    REGION: str
+    RESULT_EXTENDED: str
+    PROFILE: str
+    ACCOUNT_ID: int
+    PROVIDER: str
+    CHECKID: str
+    CHECKNAME: str
+    CHECKTITLE: str
+    CHECKTYPE: str
```

Comment:
Why all of these fields are declared in uppercase?

Reply:
At the beginning it was due to I thought it was necessary, but now it is only to help match with CSV Headers

Reply:
Declared in lower case right now